app/controllers/people_skills_controller.rb

```diff
@@ -31,4 +31,20 @@ def extract_required_search_values
     filtered_array = filter_params.skill_ids.zip(filter_params.levels, filter_params.interests)
     filtered_array.reject { |arr| arr.first == '' }.transpose
   end
+
+  def find_skill_title_by_id(id)
+    Skill.find(id).title
+  end
+
+  def get_no_match_translation(index)
+    if index.zero? && index == filter_params.rows_count - 1
```

Comment:
I think you can improve this whole method a lot, this ties in with the outsourcing comment i left in the view. 
Try to split this up into several methods, use variables that speak for themselves etc.

For example `filter_params.rows_count` should be assigned to a variable especially because you use it multiple times but also to make it readable. Something like `skills_searched` speaks volumes.